app/grandchallenge/workstation_configs/forms.py

```diff
@@ -84,6 +86,18 @@ def __init__(self, *args, read_only=False, **kwargs):
             ),
         )
 
+        self.fields["overlay_segments"].help_text += format_lazy(
+            ". If a categorical overlay is shown, "
+            "it is possible to show toggles to change the visibility of the different overlay categories. "
+            "To do so, configure the categories that should be displayed. Data from the "
+            "algorithm's output.json can be added as an extra label to each "
+            "toggle using jinja templating. "
+            'Example usage: [{{"name": "background", "visible": true, "voxel_value": 0}},'
+            '{{"name": "tissue", "visible": true, "voxel_value": 1}}]. '
```

Comment:
The example no longer contains the jinja templating example. Is that on purpose?

Reply:
Good point. Since this help_text already have an example, maybe it does not need to be changed to the default one which also keep the jinja example. I will ask Miriam on the pitch.

Reply:
Yes, or incorporate the jinja example into the default help_text? It is relevant in all places, or is it not?